templates/Tests.elm

```diff
@@ -0,0 +1,14 @@
+module Tests where
+
+import ElmTest.Assertion as A exposing (assertEqual, assert)
```

Comment:
Seems reasonable to expose `assert` and `assertEqual` by default, but I think `as A` is more of a personal preference, and might surprise people.